login.js

```diff
@@ -1,23 +1,31 @@
+let isAMatch;
+
 function verifiyPassword(hash) {
-  checkpw("12345678", hash, displayResult, null);
-  return;
+  const passwords = ['greenbear757', 'organicswan134', 'bluemouse519', 'heavytiger886', 'heavyladybug575'];
+
+  isAMatch = [];
+
+  passwords.forEach(
+    (item) => checkpw(item, hash,
+    (result) => {return isAMatch.push(result);}, null));
```

Comment:
Yep, `push()` was the method to use here :+1: 
Also `forEach()`, well done :+1: :+1: 

Regarding readability, I think I would rewrite it like this:

```js
  let storeResults = (result) => { return isAMatch.push(result); };
  passwords.forEach((password) => checkpw(password, hash, storeResults, null));
```

The reason is because how you have it now it makes it look like you are passing two functions to the `forEach`, and the arguments to `checkpw` are cut in the middle. It's not common practice to put some arguments to a function in one line and some arguments in another line :slightly_smiling_face: